tools/fetch_topsites.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Script for assigning priority to sites.
+
+The rule is as follows:
+    Critical: alexa top 100 in worldwide
+    Important: alexa top 101-1000 in worldwide or alexa top 100 in tier 1 countries/regions
+    Normal: alexa top 1001-10000 or alexa top 101-1000 in tier 1 countries/regions
+"""
+
+import argparse
+
+from tools.topsites.tranco import Tranco
+from tools.topsites.alexa import Alexa
+from tools.topsites.siterank import SiteRank
+from tools.topsites.siterank import DATA_PATH
+from tools.topsites.utils import parse_site_dom_element
+
+REGIONS = ['US', 'FR', 'IN', 'DE', 'TW', 'ID', 'HK', 'SG', 'PL',
+           'GB', 'RU']
```

Comment:
Another thought, this was initially created like this I guess because of priority market for Firefox, but we can imagine that other browsers could have a desire to extend this. Nothing to do now.